src/html/js/common_personium.js

```diff
@@ -308,11 +308,13 @@ Common.sendMessageAPI = function(uuid, extCell, type, title, body, reqRel, reqRe
     data.Title = title;
     data.Body = body;
     data.Priority = 3;
-    if (reqRel) {
-        data.RequestRelation = reqRel;
-    }
-    if (reqRelTar) {
-        data.RequestRelationTarget = reqRelTar;
+    if (reqType) {
```

Comment:
What happen when reqType is null?

Reply:
When reqType is null, we ignore RequestObjects because type is "message".

Reply:
I understood. The following 2 functions actually does not require the last 3 arguments.  
1. Common.approvalRel  
1. Common.rejectionRel